.gitignore

```diff
@@ -25,3 +25,4 @@ _testmain.go
 *.prof
 
 # for this repo only
+/.idea/
```

Comment:
Please remove this.
You can add this to your global gitignore file:
https://davidwalsh.name/global-gitignore